src/main/scala/io/viash/packageConfig/PackageConfig.scala

```diff
@@ -55,8 +55,20 @@ case class PackageConfig(
   @since("Viash 0.9.0")
   version: Option[String] = None,
 
+  @description("A clean notation of the package. This is used when generating documentation.")
+  @example("label: my package", "yaml")
+  @default("Empty")
+  @since("Viash 0.9.0")
+  label: Option[String] = None,
+
+  @description("A short summary of the package. This is used when generating documentation.")
+  @example("summary: My package is used for function X", "yaml")
+  @default("Empty")
+  @since("Viash 0.9.0")
+  summary: Option[String] = None,
```

Comment:
```suggestion
  @description("A clean version of the package name. This is only used for documentation.")
  @example("label: \"My package\"", "yaml")
  @default("Empty")
  @since("Viash 0.9.0")
  label: Option[String] = None,

  @description("A one-sentence summary of the package. This is only used for documentation.")
  @example("summary: \"This package is used for XYZ.\"", "yaml")
  @default("Empty")
  @since("Viash 0.9.0")
  summary: Option[String] = None,
```